bower.json

```diff
@@ -12,11 +12,11 @@
   ],
   "dependencies": {
     "angular": "~1.5",
-    "angular-ui-router": "~0.2.15",
-    "ngstorage": "~0.3.3",
-    "angular-material": "~1.1",
+    "angular-ui-router": "~1.0.0-alpha.3",
+    "ngstorage": "~0.3.10",
+    "angular-material": "~1.0.8",
```

Comment:
@GrahamCampbell  No it's not, in fact they remove every 1.1.* versions from they're web as  you can see [here](https://material.angularjs.org/latest/) this is the latest version that came just yesterday